templates/deno/remix.config.js

```diff
@@ -1,12 +1,13 @@
 module.exports = {
   serverBuildTarget: "deno",
   server: "./server.ts",
-
-  /*
-  If live reload causes page to re-render without changes (live reload is too fast),
-  increase the dev server broadcast delay.
-
-  If live reload seems slow, try to decrease the dev server broadcast delay.
-  */
+  // If live reload causes page to re-render without changes (live reload is too fast),
+  // increase the dev server broadcast delay.
+  // If live reload seems slow, try to decrease the dev server broadcast delay.
   devServerBroadcastDelay: 300,
+  ignoredRouteFiles: ["**/.*"],
+  // appDirectory: "app",
+  // assetsBuildDirectory: "public/build",
+  // serverBuildPath: "build/index.js",
+  // publicPath: "/build/",
```

Comment:
Can you include a link to the Remix config docs instead of commenting out defaults?

Reply:
I'd prefer if we linked to the Remix config docs across all of our templates. I don't think there's value in having consistency in commented out defaults across templates as long as the information is accessible. Plus the docs would explain why the defaults are what they are, and would also show all the other possible options.

Could you replace this with a link to the Remix config docs?

Reply:
That should be done in a separate PR imo.

Let's merge these changes so all templates are consistent & we can remove it again in another PR if we want to.